google-cloud-storage/lib/google/cloud/storage/file.rb

```diff
@@ -268,6 +268,35 @@ def encryption_key_sha256
         end
 
         ##
+        # The file's storage class. This defines how the file is stored and
+        # determines the SLA and the cost of storage. For more information, see
+        # [Storage
+        # Classes](https://cloud.google.com/storage/docs/storage-classes) and
+        # [Per-Object Storage
+        # Class](https://cloud.google.com/storage/docs/per-object-storage-class).
+        def storage_class
+          @gapi.storage_class
+        end
+
+        ##
+        # Updates how the file is stored and determines the SLA and the cost of
+        # storage. Values include `:multi_regional`, `:regional`, `:nearline`,
+        # `:coldline`, `:standard`, and `:dra` (Durable Reduced Availability),
+        # as well as the strings returned by {File#storage_class} or
+        # {Bucket#storage_class}. For more information, see [Storage
+        # Classes](https://cloud.google.com/storage/docs/storage-classes) and
+        # [Per-Object Storage
+        # Class](https://cloud.google.com/storage/docs/per-object-storage-class).
+        # The  default value is the default storage class for the bucket. See
+        # {Bucket#storage_class}.
+        # @param [Symbol, String] storage_class Storage class of the file.
+        def storage_class= storage_class
```

Comment:
I didn't add one because the other setter methods don't have examples, or return types. I didn't want this method to stand out from the rest. But yes, they should have them.